index.js

```diff
@@ -58,7 +63,7 @@ function getBinding (type, client, id) {
       try {
         const bindingType =
           fs.readFileSync(path.join(root, file, 'type')).toString().trim();
-        if (bindingType === typeMapping[type]) {
+        if (bindingType === typeMapping[type] || aliases[typeMapping[type]].includes(bindingType)) {
```

Comment:
@pacostas we will need to add some additional tests for this, including making sure things are ok when there is no alias.  I think there will need to be more checks otherwise the includes () may throw when there is no mapping.